docs/docs/hardware.mdx

```diff
@@ -6,13 +6,33 @@ sidebar_label: Supported Hardware
 import HardwareList from "@site/src/components/hardware-list";
 import Metadata from "@site/src/data/hardware-metadata.json";
 
+import { groupedMetadata } from "@site/src/components/hardware-utils";
+
+export const toc = [
+  {
+    value: "Onboard Controller Keyboards",
+    id: "onboard",
+    level: 2,
+  },
+  {
+    value: "Composite Keyboards",
+    id: "composite",
+    level: 2,
+  },
+  ...Object.values(groupedMetadata(Metadata).interconnects).map(
+    ({ interconnect }) => ({
+      value: `${interconnect.name} Interconnect`,
+      id: interconnect.id,
+      level: 3,
+    })
+  ),
+];
+
 With the solid technical foundation of Zephyr™ RTOS, ZMK can support a wide diversity of hardware targets.
 That being said, there are currently only a few specific [boards](/docs/faq#what-is-a-board)/[shields](faq.md#what-is-a-shield) that have been implemented and tested by the ZMK contributors.
 
 <HardwareList items={Metadata} />
 
-## Other Hardware
-
```

Comment:
Has this heading been intentionally removed? 

Hierarchically, this _Other Hardware_ (and _Contributing_) content now sits under the "Seeed XIAO Interconnect" heading:

---

<img width="517" alt="Screenshot 2022-04-18 at 6 46 56 am" src="https://user-images.githubusercontent.com/60922914/163761131-4e321844-c1c1-43fa-acf0-a21179effed8.png">

---

I think it would be nice to have these two sections headed and part of the ToC if possible?

Reply:
Yeah, accidentally removed this, was unintentional. Working on fixing.

Reply:
@dxmh Ok, restored those headings, and cleaned this up a bit so the headers are 100% properly styled in the generated MDX stuff by using the component from `@theme/Heading` directly everywhere.